q/gradle-quickstart.sh

```diff
@@ -41,7 +41,7 @@ function mkPackage() {
 defaultProjectName="quickstart"
 defaultOrganization="org.myorg.quickstart"
 defaultVersion="0.1-SNAPSHOT"
-defaultFlinkVersion="${1:-1.12.1}
+defaultFlinkVersion="${1:-1.12.2}
```

Comment:
This line is missing an ending quote and will cause the script to not work. I had entered a pull request for the 1.12.1 version to fix that.

Change this `defaultFlinkVersion="${1:-1.12.2}` to this `defaultFlinkVersion="${1:-1.12.2}"`

Reply:
Thanks!